packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -184,6 +195,34 @@ function getSaveInfo(
 	return { saveOnFailure: true, filepath };
 }
 
+export class DDSFuzzHandle<T = IFluidLoadable & FluidObject> implements IFluidHandle {
```

Comment:
What object fulfills the requirements of IFluidLoadable and FluidObject? I assume I would want a more specific type here, but I'm not sure which object that would be

Reply:
Look at blobhandle in blob manager. i would model this after that, so it just points to content, so get will probably just return absolute path for get. Then there is the other part, which is resolving the handle, take a look at resolveHandle in containerRuntime. i think the mock container runtime will have  to be updated to understand the path in resolve handle, and return the same same thing so that on round trip the handles can be resolved.

Reply:
might also be good to put this in its own file, as this file is getting huge

Reply:
on round trip, so a dds receiving an op with a handle will decode it as one of these: packages\dds\shared-object-base\src\remoteObjectHandle.ts

That will then call into the routeContext. i'm not sure exactly what that will be, probably the MockDataStoreRuntime, or the MockContainerRuntime. It doesn't really matter which object it is, but that object will be the one to resolve the handle, which for this case probably just means returning the absolute path as the resolved value